spec/lib/orcid/add_works_spec.rb

```diff
@@ -3,17 +3,25 @@
 describe Orcid::AddWorks do
   let(:add_works) { described_class.new(logger: logger) }
 
-  let(:logger) { instance_double(Logger, info: nil, error: nil) }
+  let(:logger) { instance_double(Logger, info: nil, error: nil, warn: nil) }
 
   let(:orcid_user) do
     Mais::Client::OrcidUser.new(author.sunetid, author.orcidid, ['/read-limited', '/activities/update', '/person/update'],
                                 '91gd29cb-124e-5bf8-1ard-90315b03ae12')
   end
 
+  # This is an author that has no currently valid publications that can be pushed to ORCID
+  let(:orcid_user_no_orcid_approved_contributions) do
+    Mais::Client::OrcidUser.new(author_no_orcid_approved_contributions.sunetid, author_no_orcid_approved_contributions.orcidid,
+                                ['/read-limited', '/activities/update', '/person/update'], '91gd29cb-124e-5bf8-1ard-90315b03ae12')
+  end
+
   describe '#add_for_orcid_user' do
```

Comment:
I discovered some of the existing tests were passing because the tokens were invalid and that was allowing them to pass.  This new data setup ensures they pass for the reasons the reasons stated in the test